lib/u3d/installation.rb

```diff
@@ -163,6 +167,39 @@ def plist
     end
   end
 
+  class LinuxInstallationHelper
+    STRINGS_FULL_VERSION_MATCHER = /^[0-9\.abfp]+_[0-9a-f]{12}/
```

Comment:
Here I would have to check if we get weird values, e.g. with some of the versions that had weird build numbers.